examples/script.js

```diff
@@ -5,4 +5,17 @@ export default function () {
   console.log(exec.command("ls",["-a","-l"], {
```

Comment:
This command fails and stops the execution of the script, so the new example introduced below is never executed.

I suggest putting it after the new example and adding a `console.log("this will not be printed") statement at the end of the script to show that without the `continue_on_error` option, any error stops the script.

See example below:

<details>
<summary> click to show script</summary>

```js
import exec from 'k6/x/exec';

export default function () {
  console.log(exec.command("date"));

  // With custom error handling
  try {
    var output = exec.command("ls",["-a", "NO_SUCH_DIR"], {
      "continue_on_error": exec
    });
  } catch (e) {
        console.log("ERROR: " + e);
        if (e.value && e.value.stderr) {
                console.log("STDERR: " + String.fromCharCode.apply(null, e.value.stderr))
        }
  }

  // without error handling the test will stop when the following command fails
  console.log(exec.command("ls",["-a","-l"], {
     "dir": "sub-directory" // optional directory in which the command has to be run
  }));

  console.log("this message will not be printed")
}
```
</details>

This script produces the following output:
<details>
<summary> click to see output</summary>

```sh
          /\      |‾‾| /‾‾/   /‾‾/   
     /\  /  \     |  |/  /   /  /    
    /  \/    \    |     (   /   ‾‾\  
   /          \   |  |\  \ |  (‾)  | 
  / __________ \  |__| \__\ \_____/ .io

     execution: local
        script: script.js
        output: -

     scenarios: (100.00%) 1 scenario, 1 max VUs, 10m30s max duration (incl. graceful stop):
              * default: 1 iterations for each of 1 VUs (maxDuration: 10m0s, gracefulStop: 30s)

INFO[0000] mar 09 jul 2024 12:12:50 CEST                 source=console
INFO[0000] ERROR: GoError: exit status 2                 source=console
INFO[0000] STDERR: ls: cannot access 'NO_SUCH_DIR': No such file or directory  source=console
INFO[0000] 2024/07/09 12:12:50 chdir sub-directory: no such file or directory on command: ls -a -l 
```

</details>

Reply:
Well spotted, thanks :) Changes pushed, fyi